.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -95,8 +95,14 @@ def load_results(results_bucket: str,
     print("Loading existing accumulative results ...")
     accumulative_results = {}
     try:
-        content = util.download_blob_into_memory(results_bucket, results_file, download_as_text=True)
-        accumulative_results = json.loads(content)
+        client = storage.Client()
+        bucket = client.bucket(results_bucket)
+
+        build_results_dir = os.path.dirname(results_file)
+        blobs = client.list_blobs(results_bucket, prefix=build_results_dir)
+        for blob in blobs:
+            content = util.download_blob_into_memory(results_bucket, blob.name, download_as_text=True)
+            accumulative_results = {**accumulative_results, **json.loads(content)}
```

Comment:
Strange but I guess you'll be replacing with a list of `NotebookExecutionResults`?

I'm not understanding what this line does.

Reply:
Each build outputs its own results file (ie., {build_id}.json) into the same GCS directory (../build_results). I scan the directory, load each JSON result file and then merge it with accumulative results.

For example, let's say notebook foo.ipynb appears in three {build_id}.json files, with 2 passes and 1 fail. Then accumulative results would look like:

{'foobar.ipynb': {'passed': 2, 'failed': 1, .... }